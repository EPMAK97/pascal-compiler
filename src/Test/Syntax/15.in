const
    one = 1;
    two = 2.3 + 2;
var
    a, b : integer;
    c, d : double;
type
    a = integer;
    c = a;
    c, m, n = array [1..10] of integer;
Begin