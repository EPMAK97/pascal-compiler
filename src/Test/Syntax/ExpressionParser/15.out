Error in pos 1:7 expected identifier, constant or expression 
